BridgeAppSDK/SBABridgeInfo.swift

```diff
@@ -91,6 +91,32 @@ public protocol SBABridgeInfo: SBASharedAppInfo, SBBBridgeInfoProtocol {
      do not check for a text user.
     */
     var disableTestUserCheck: Bool { get }
+    
+    /**
+     Show a back button in the navigation bar of ORKTaskViewController.
+     */
+    var showBackButtonInNavBar: Bool { get }
```

Comment:
The use of the plist for the BridgeInfo started as a work-around for AppCore and BridgeAppSDK not knowing about each other. When we split out some of the functionality into the factory framework and BridgeSDK, it allowed that flexibility.

That said, I'd like to move away from new development using this file.  It's useful for setting up the server config to allow us to extend what is supported by the server model, but it's not used for UI and I'd like to keep it agnostic to ResearchKit. :)